src/__tests__/render.js

```diff
@@ -3,6 +3,13 @@ import ReactDOM from 'react-dom'
 import ReactDOMServer from 'react-dom/server'
 import {fireEvent, render, screen, configure} from '../'
 
+// Needs to be changed to 19.0.0 once alpha started.
+const isReactExperimental = React.version.startsWith('18.3.0-experimental')
+const isReactCanary = React.version.startsWith('18.3.0')
```

Comment:
Shouldn't we check for `React.version.startsWith('18.3.0-canary')` here?

Reply:
yep